SQL/New_patches/2019-10-29_adding_issues_attachments_table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE `issues_attachments` (
```

Comment:
The default schema file `SQL/0000-00-00-schema.sql` needs to be updated as well.

Reply:
Thanks @johnsaigle